code.json

```diff
@@ -42,6 +42,52 @@
       "email": "jlaura@usgs.gov"
     },
 
+    "date": {
+      "metadataLastUpdated": "2023-04-07"
+    }
+  }, {
+    "name": "ale",
+    "organization": "U.S. Geological Survey",
+    "description": "GitLab repository for the Abstraction Layer for Ephemerides package",
+    "version": "0.8.8",
+    "status": "Development",
+
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://code.usgs.gov/astrogeology/ale/-/raw/0.8.8/LICENSE.md"
+        }
+      ]
+    },
+
+    "homepageURL": "https://code.usgs.gov/astrogeology/ale/0.8.8",
+    "downloadURL": "https://code.usgs.gov/astrogeology/ale/-/archive/0.8.8/ale-main.zip",
+    "disclaimerURL": "https://code.usgs.gov/astrogeology/ale/-/raw/0.8.8/DISCLAIMER.md",
+    "repositoryURL": "https://code.usgs.gov/astrogeology/ale.git",
+    "vcs": "git",
+
+    "laborHours": 480,
+
+    "tags": [
+      "Planetary",
+      "Remote Sensing",
+      "Data Processing",
+      "Ephemerides",
+      "Community Sensor Model"
+    ],
+
+    "languages": [
+      "C++",
+      "Python"
+    ],
+
+    "contact": {
+      "name": "Jason Laura",
+      "email": "jlaura@usgs.gov"
```

Comment:
This should be you, no? I was just in the last one because I did this part of the release.